content/vault/v1.20.x/content/docs/updates/important-changes.mdx

```diff
@@ -400,3 +400,16 @@ filters you have two options:
 
 1. Spread them out among the nodes of the Vault cluster.
 1. Only subscribe to events on the active node of the cluster.
+
+### Full seal rewraps occur on DR/PR failover with multi-seal enabled ((#multi-seal-rewrap)) <EnterpriseAlert inline="true" />
```

Comment:
Same question about the partial

Reply:
Addressed within https://github.com/hashicorp/web-unified-docs/pull/1000/commits/5252f4e73f217649f69dd7bc9c5d5d79af1d8f7f